admin/listPurchases/listView.html

```diff
@@ -298,7 +298,7 @@
 async function requestOrders(callback) {
     let response;
     try{
-        response = await fetch("/admin/orders", {
```

Comment:
The complexity of this line is really high. Extract the `URLSearchParams` to a separate variable to make it easier to understand and reason about.